static/app/views/settings/featureFlags/index.tsx

```diff
@@ -164,6 +165,7 @@ export function OrganizationFeatureFlagsIndex() {
         )}
       </TextBlock>
 
+      <h5>Providers</h5>
```

Comment:
`GridEditable` should have a built in `title` prop so we could match that styling here (you could even update the `PanelTable` to include a similar title prop), and pass in a title for your audit log table. 

otherwise, let's translate the title

```suggestion
      <h5>t('Providers')</h5>
```

Reply:
Gonna translate. Is it fine to leave as h5's? Since for 'Audit Logs' I'm putting it in a Flex

Reply:
yeah `h5` is fine